store/src/app/products/components/cart/cart.component.html

```diff
@@ -0,0 +1,29 @@
+<ng-container *ngIf="products.length >= 1">
+  <div class="container">
+    <h1 class="title">Cart</h1>
+    <p>Total: {{totalPrice | hryvniaPipe}}</p>
+    <div class="products-cards">
+      <app-product-card [quantityButtons]="true" *ngFor="let item of products" [product]="item"
+        [cartButtonName]="cartButtonName">
+      </app-product-card>
+      <br>
+    </div>
+  </div>
+</ng-container>
+
+<ng-container *ngIf="products.length === 0 || products === null">
```

Comment:
Could be simplified like

`*ngIf="!products?.length"`

Reply:
Done